pyfolio/timeseries.py

```diff
@@ -584,6 +596,40 @@ def rolling_multifactor_beta(returns, df_multi_factor, rolling_window=63):
     return out
 
 
+def rolling_fama_french(returns, risk_factors=None,
```

Comment:
Like this workflow a lot more, robust to people using their own factors now :+1: